pkg/printers/internalversion/describe.go

```diff
@@ -1136,6 +1136,12 @@ func describePersistentVolume(pv *api.PersistentVolume, events *api.EventList) (
 		}
 		w.Write(LEVEL_0, "Reclaim Policy:\t%v\n", pv.Spec.PersistentVolumeReclaimPolicy)
 		w.Write(LEVEL_0, "Access Modes:\t%s\n", helper.GetAccessModesAsString(pv.Spec.AccessModes))
+		if pv.Spec.VolumeMode != nil {
+			w.Write(LEVEL_0, "VolumeMode:\t%v\n", *pv.Spec.VolumeMode)
+		} else {
+			var volumeMode api.PersistentVolumeMode
+			w.Write(LEVEL_0, "VolumeMode:\t%v\n", volumeMode)
```

Comment:
omit this else block entirely, right? this prints out an empty string as is

Reply:
done.